src/component/SelectedCards/SelectedCards.jsx

```diff
@@ -0,0 +1,69 @@
+import { useEffect, useState } from "react";
+import { Checkbox, Typography } from "@material-ui/core";
+import { Box, Button, Card, CardContent } from "../../style/style";
+import { NavLink } from "react-router-dom";
+
+const SelectedCards = (props) => {
```

Comment:
it is better  to use destructure of the props everywhere   
   SelectedCards = ({ selectedCards deleteCountryCard,}) => {